awx/ui/src/screens/InstanceGroup/InstanceGroups.js

```diff
@@ -16,31 +17,28 @@ import ContainerGroup from './ContainerGroup';
 
 function InstanceGroups() {
   const { pathname } = useLocation();
+  const { isSuperUser, isSystemAuditor } = useUserProfile();
+  const userCanReadSettings = isSuperUser || isSystemAuditor;
+
   const {
     request: settingsRequest,
     isLoading: isSettingsRequestLoading,
     error: settingsRequestError,
-    result: { isKubernetes, defaultControlPlane, defaultExecution },
+    result: { isKubernetes },
   } = useRequest(
     useCallback(async () => {
       const {
-        data: {
-          IS_K8S,
-          DEFAULT_CONTROL_PLANE_QUEUE_NAME,
-          DEFAULT_EXECUTION_QUEUE_NAME,
-        },
+        data: { IS_K8S },
```

Comment:
What happens here is an org admin accesses instance groups list on a kubernetes deployment?

Reply:
This user profile can not read settings endpoint. This variable is used used to enable or not the add button for `instance groups` and `container groups` and this user will not have the privilege to add those not matter what. Then `IS_K8S` will default to initial value to false, and we do not make a extra requests to settings, and also do display the add button.